finalPrep/annotations.js

```diff
@@ -2,41 +2,67 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// This is a function with 5 parameters (name, age, fur, clothers and special Powers)
 function buildABear(name, age, fur, clothes, specialPower) {
+  //Declares the varible  greeting. It is a string with name interpolated in it.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  // Declares the varible demographics. It is an array with name and age
   var demographics = [name, age];
+  // declares the varible powerSaying. It is string with concatenation two strings with the varible specialPower
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  // Declares the varible builtBear with 5 key value pairs
   var builtBear = {
+//Keyvalue pair basicInfo with demographics varible
     basicInfo: demographics,
+//Keyvalue pairs clothes clothes with clothes
     clothes: clothes,
```

Comment:
I am liking the technical terminology, like declaring the variable, and invoking the function. For your object, I may make the key versus the value a bit more clear, perhaps like 'assigns a key named 'clothes' to the value clothes passes in as a param.'